presto-main/src/main/java/com/facebook/presto/execution/CreateTableTask.java

```diff
@@ -128,7 +129,11 @@ public String explain(CreateTable statement, List<Expression> parameters)
                         metadata,
                         parameters);
 
-                columns.put(name, new ColumnMetadata(name, type, column.getComment().orElse(null), null, false, columnProperties));
+                Optional<Object> maybeDefaultValue = column.getDefaultValue()
```

Comment:
`s/maybeDefaultValue/defaultValue` (by convention, the `Optional` type conveys the "maybe-ness")